daemon/api_prompting.go

```diff
@@ -441,6 +450,9 @@ func postRules(c *Command, r *http.Request, user *auth.UserState) Response {
 		return promptingNotRunningError()
 	}
 
+	// TODO: check if it's a handler service client making the API request
+	clientActivity := false
```

Comment:
Why is this false here?

Regarding the TODOs above, it might be worth having a function that takes some information, holds the TODO and returns true for now, to have one place to fix up later.

Having said that I'm not sure about why one is true and one is false, so perhaps this does not make sense.

Reply:
This and the one below are false because these are GET/POST on the rules endpoints, as opposed to the prompts endpoints. Prompt handler services are designed to listen for and respond to prompts, while other clients like the Security Center may be reading rules without actually being available as a prompt handler.

As for whether we should have a placeholder helper function, I think that's a good idea, though I don't yet know what we're going to be checking to know whether the client is a handler service daemon, and we do in the meantime need to differentiate prompts endpoints from rules endpoints.

Perhaps the best solution would be to add the helper and call it from all of the prompts endpoint handlers, and leave `clientActivity := false` on the rules endpoints, since I don't think we'll ever want to treat activity on the rules endpoints as client activity.